src/main/drivers/pwm_mapping.h

```diff
@@ -36,10 +36,10 @@
 #define MAX_INPUTS  8
 
 #define PWM_TIMER_MHZ 1
-//these three have to be the same because of the ppmAvoidPWMTimerClash functions
-#define ONESHOT_TIMER_MHZ 12
+#define ONESHOT125_TIMER_MHZ 8
+#define ONESHOT42_TIMER_MHZ 24
```

Comment:
No, it has to be under clocked to 96 MHz to work with USB at 48 MHz. This allows it to work with the 12 MHz timer.